vr-wg-charter.html

```diff
@@ -373,23 +368,21 @@ <h3 id="w3c-coordination">
           </h3>
           <dl>
             <dt>
-              <a href="https://www.w3.org/community/webvr/">WebVR Community
+              <a href="https://www.w3.org/community/webvr/">Immersive Web Community
               Group</a>
             </dt>
             <dd>
-              The WebVR Community Group will provide the WebVR API "v2.0" seed
-              specification to begin the standards process. The WebVR API v1.1
+              The Immersive Web Community Group will provide the WebXR Device API seed specification to begin the standards process. The WebVR API v1.1
               maintenance is expected to continue in parallel in the Community
```

Comment:
Do we want to note WebXR Device API new feature incubation (i.e. features that'd be out of scope for the WG) will happen in the CG?